app/src/androidTest/java/com/eis/audioUtility/AudioUtilityManagerTest.java

```diff
@@ -0,0 +1,202 @@
+package com.eis.audioUtility;
+
+/**
+ * @author Francesco Bau', Alberto Ursino
+ */
+
+import android.content.Context;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AudioUtilityManagerTest {
+
+    Context context;
+
+    @Before
+    public void init() {
+        context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+    }
+
+    @Test
+    public void setAlarmVolume_percentage_isTooLow() {
+        try {
+            AudioUtilityManager.setVolume(context, AudioUtilityManager.AUMStream.ALARM, -1);
+            Assert.fail("Should have thrown IllegalArgumentException.");
+        } catch (IllegalArgumentException e) {
+            //Success
```

Comment:
IllegalArgumentException is a runtime exception, you can remove all of this an keep the `AudioUtilityManager.setVolume(...)` line and add on top of the method `@Test(excepted = IllegalArgumentException.class)`, it will do this exact same job

Reply:
Ok. Thank you for the advice.